# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1337364327 25200
#      Fri May 18 11:05:27 2012 -0700
# Node ID 8f792cff711ecf176f426d7dfc1b11e2eb017bc0
# Parent  5fa6a41d4fbe73d69f4290631f26f3371ca57215
build: Distribute macros.texi in tarball so documentation will build.

* interpreter/Makefile.am: Add macros.texi to EXTRA_DIST variable.

diff --git a/doc/interpreter/Makefile.am b/doc/interpreter/Makefile.am
--- a/doc/interpreter/Makefile.am
+++ b/doc/interpreter/Makefile.am
@@ -240,16 +240,17 @@ spellcheck: $(SPELLCHECK_FILES)
 
 EXTRA_DIST = \
   config-images.sh \
   contributors.in \
   find-docstring-files.sh \
   images \
   images.awk \
   images.mk \
+  macros.texi \
   mk_doc_cache.m \
   mkcontrib.awk \
   munge-texi.pl \
   octave.dvi \
   octave.html \
   octave.pdf \
   octave.ps \
   $(IMAGES) \
